apertium-eng.eng.rlx

```diff
@@ -212,6 +217,54 @@ SELECT N IF (0 N) (0 Adj) (1 V) ;
 
 SELECT Adj IF (0 N) (0 Adj) (1 N) ;
 
+# Different cases for as ("as __ as")
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-3 Vbser) ;
+#Example (as):
+#You are as quick as I am.
+
+SELECT Pr IF (0 As) (-2 Preadv) (-1 Adv) (-4 Vbser) ;
+#Example (as):
+#The Earth was formed as early as 4.54 billion years ago.
```

Comment:
This is less about `-4` being `Vbser` and more about `-3` being a verb, no?

Reply:
I've addressed this.